include/nlohmann/ordered_map.hpp

```diff
@@ -122,7 +122,7 @@ template <class Key, class T, class IgnoredLess = std::less<Key>,
     {
         for (auto it = this->begin(); it != this->end(); ++it)
         {
-            if (m_compare(it->first, key))
+            if (m_compare(it->first, std::forward<KeyType>(key)))
```

Comment:
I'm not sure these are valid, since it's in a loop.  We don't want to move from this more than once.  (`std::equal_to.operator()` doesn't move, so that's not an issue with the default, but could be with another function.)

Reply:
I'll check.